doc/fmt.less

```diff
@@ -41,8 +41,8 @@ div.sphinxsidebar {
   text-align: center;
 }
 
-// Indent descriptions of classes, functions and macros.
-.class dd, .function dd, .macro dd {
+// Indent descriptions of classes, functions, macros, and structs.
+.class dd, .function dd, .macro dd, .struct dd {
```

Comment:
Please move stylesheet changes to a separate PR and provide some details why they are an improvement e.g. before/after screenshots.